index.html

```diff
@@ -0,0 +1,178 @@
+
+        </div>
+        <div id="whiteHeader">
+            <a class="logoLink" href="./index.html"><img id="logoHeader" src="./images/logoOrangeMx-removebg-preview.png" alt="logo"></a>
+            <div id="menuOptions">
+                <a class="menuOptionsLinks" href="./about.html">
+                    <div>About</div>
+                </a>
+                <a class="menuOptionsLinks" href="./about.html">
+                    <div>Event</div>
+                </a>
+                <a class="menuOptionsLinks" href="#programDetails">
+                    <div>Join</div>
+                </a>
+                <a class="menuOptionsLinks sponsorMobile" href="./about.html#footerOne">
+                    <div>Sponsor</div>
+                </a>
+                <a class="menuOptionsLinks sponsorDesk" href="#footerOne">
+                    <div>Sponsor</div>
+                </a>
+                <a class="menuOptionsLinks" href="#">
+                    <div>News</div>
+                </a>
+                <a class="menuOptionsLinks ccCampaign" href="#">
+                    <div>CC Campaign</div>
+                </a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <!-- Main Section -->
+        <section class="main">
+                <div id="mobileTopContainer">
+                    <nav class="hamburgerMenu">
+                        <span class="bar bar1"></span>
+                        <span class="bar bar2"></span>
+                        <span class="bar bar3"></span>
+                    </nav>
+                    <div id="firstMain"></div>
+                </div>
+                <!-- ---- -->
+                <img class="adornIcon taco" src="./images/iconsMxImages/6bigTaco-removebg-preview.png" alt="">
+                <img class="adornIcon hat" src="./images/iconsMxImages/2hat-removebg-preview.png" alt="">
+                <img class="adornIcon pinata" src="./images/iconsMxImages/3pin-removebg-preview.png" alt="">
+                <img class="adornIcon guacamole" src="./images/iconsMxImages/7guacamole-removebg-preview.png" alt="">
+                <!-- --- -->
+                <div id="textMain">
+                    <div id="helloAndEventContainer">
+                        <div id="hello">"Hello! Food Lovers"</div>
+                        <h1 id="event">INTERNATIONAL MEXICAN FOOD CONVENTION 2023</h1>
+                    </div>
+                    <div id="description">
+                        <p id="descriptionParagraph">A joyful celebration believing in the value of openness and
+                            sharing, creating a positive change with people from over 80 countries is taking place in
+                            October, in Korea.</p>
+                    </div>
+                    <div id="dateAndPlaceContainer">
+                        <div id="date">2023.10.15(THU) ~ 16(FRI)</div>
+                        <div id="place">@ National Museum of Karea, Art Center Nabi and more</div>
```

Comment:
- [optional] kindly use `h` tags here or `p` tags. `div's` are used for defining a section or a division in an HTML page.